Tests/CommonHelpers/Acts/Tests/CommonHelpers/MeasurementsCreator.hpp

```diff
@@ -26,6 +26,7 @@
 namespace Acts {
 namespace Test {
 
+using namespace Acts::detail::Test;
```

Comment:
`using namespace` is dangerous. Can this be avoided?

Reply:
I indeed uncovered hidden depency on this namespace in several source files. Fixed now.